src/components/CircularProgressBar/CircularProgressBar.js

```diff
@@ -117,7 +117,7 @@ const CircularProgressBar = ({
       <AnimatedPath
         d={path}
         fill="transparent"
-        stroke={getThemeType(theme) === DARK_THEME ? colors.tertiary : '#F7F9FB'}
+        stroke={getThemeType(theme) === DARK_THEME ? colors.basic070 : '#F7F9FB'}
```

Comment:
Just checking, did you mean for the progress bar to disappear in the background? I was looking at these screens for reference: https://app.zeplin.io/project/5f4e5cdde148ba0c5d384e07/screen/5f6b75b4feedf54da55cdc8e, https://app.zeplin.io/project/5f4e5cdde148ba0c5d384e07/screen/5f6b75bae311552df818411a and they have a slightly different bar colour.